src/plugin/cursus/Resources/config/config.yml

```diff
@@ -21,6 +21,11 @@ plugin:
           context: [desktop]
           tags: [community, event]
 
+        - name: all_courses
+          type: list
+          context: [workspace, desktop, home]
```

Comment:
It shouldn't be available in `workspace` context.

Reply:
I'm removed workspace